src/frontend/styles/components/sidebar.pcss

```diff
@@ -26,8 +26,10 @@
       display: none;
     }
   }
-
-
+  &__menu-wrapper {
+    overflow: auto;
```

Comment:
this change leads to incorrect scrollbar position
<img width="388" alt="image" src="https://user-images.githubusercontent.com/31101125/174967983-adb6acbe-f495-4a95-9f29-bef3e7bc01e4.png">

scrollbar should be positioned as follows (screenshot from stage):
<img width="345" alt="image" src="https://user-images.githubusercontent.com/31101125/174967736-53c73ecf-fa55-4dc7-85a8-bc88f0017a5d.png">

Reply:
if this change is made in order to stick CodeX logo to the bottom of the screen then you should know that we decided not to stick the logo as it distracts user from sidebar content